application/modules/invoices/views/modal_copy_invoice.php

```diff
@@ -12,6 +12,7 @@
         $('#copy_invoice_confirm').click(function () {
             show_loader(); // Show spinner
             $.post("<?php echo site_url('invoices/ajax/copy_invoice'); ?>", {
+                    legacy_calculation: $('#btn_copy_invoice').data('legacy-calculation'),
```

Comment:
needs translation

Reply:
Explanation:
Let's imagine that this configuration is set to `true` by default by the config `init` and that the customer needs an e-invoicel with `legacy_calculation` `false` (_automatic mode now_). When the copy is made, it will be with the `true` mode without this (it will have other vat amounts).

It's for simplify and to avoid cluttering ajax controllers. (otherwise it doesn't know the calculation mode).

Reply:
Resolved by
[[eInvoice] DRY: Use legacy_calculation constant in modal scripts](https://github.com/InvoicePlane/InvoicePlane/pull/1272/commits/8ff6a42a26a30cd5d579a3a23697965f27a98016)